### DISCUSSION TITLE
Magisk 26.1 doesn't work?

### DISCUSSION THREAD

**Discussion by Ognisty321** [LSPosed/MagiskOnWSALocal#486] [General]
Hey, I tried to install WSA with newest Magisk (WSA_2302.40000.9.0_x64_Release-Nightly-with-magisk-26.1(26100)-stable-MindTheGapps-13.0), but it seems that Magisk isn't even installed. Anyone?

![EfqJ682 1](https://user-images.githubusercontent.com/32206254/231189198-33f622bd-ce1a-426a-9b06-48f82073c306.png)

**Comment by MustardChef** [comment]
Opened an issue regarding this problem

**Reply by Ognisty321** [reply]
Thanks. There was too much things to fill, so I didn't bother to open an issue
